src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskConversionTest.java

```diff
@@ -259,7 +269,7 @@ public void testNullableDate() throws Exception {
   @Test
   public void testTime() throws Exception {
     GregorianCalendar expected = new GregorianCalendar(1970, Calendar.JANUARY, 1, 23, 3, 20);
-    expected.setTimeZone(TimeZone.getTimeZone("UTC"));
+    expected.setTimeZone(timezone);
```

Comment:
Note that these testcases were explicitly validating the objects were in UTC time. They failed once I updated them to test with non-UTC timezone.